examples/gpgpu/opencl.c

```diff
@@ -5,16 +5,16 @@
 
 /*@
     context_everywhere opencl_gcount == 1;
-    context \pointer_index(a, get_local_id(0)*2, write);
-    context \pointer_index(a, get_local_id(0)*2+1, write);
-    context \pointer_index(b, get_local_id(0), write);
+    context \pointer_index(a, \ltid*2, write);
```

Comment:
Because it's annoying to write out every time :). In larger examples you get huge expressions like `a[get_local_id(0)] == b[get_local_id(0)] + c[get_local_id(0)]`